controlplane/src/core/repositories/FederatedGraphRepository.ts

```diff
@@ -456,15 +637,12 @@ export class FederatedGraphRepository {
         routingUrl: fedGraph.routingUrl,
         subgraphsCount: fedGraph.subgraphsCount,
         composedSchemaVersionId: insertedVersion[0].insertedId,
+        namespace: fedGraph.namespace,
+        namespaceId: fedGraph.namespaceId,
       };
     });
   }
 
-  public async exists(name: string) {
```

Comment:
I wrote a proper implementation that only checks if the graphs exists.

Reply:
Updated